sdk/typescript/package.json

```diff
@@ -21,20 +21,20 @@
     "@lifeomic/axios-fetch": "^3.0.1",
     "@opentelemetry/api": "^1.8.0",
     "@opentelemetry/exporter-trace-otlp-grpc": "^0.51.0",
-    "@opentelemetry/sdk-metrics": "^1.23.0",
+    "@opentelemetry/sdk-metrics": "^1.24.0",
     "@opentelemetry/sdk-node": "^0.51.0",
-    "@opentelemetry/semantic-conventions": "^1.23.0",
+    "@opentelemetry/semantic-conventions": "^1.24.0",
```

Comment:
Hm, with bumping these, I would say to sanity check that this keeps things working.

I think it should be fine, but changing the OTEL semantic conventions in only one place before has been an *utter* pain in the past. That said, go currently uses 1.24, so it *should be fine* (still worth confirming though).

Reply:
I'm going to check with a repro to verify then

Reply:
LGTM: https://dagger.cloud/Quartz/traces/82db2929cd258fe2637eba3f663fc6ce

Reply:
Thanks for checking! :tada: